receiver/dcgmreceiver/client.go

```diff
@@ -66,15 +70,21 @@ func newClient(config *Config, logger *zap.Logger) (*dcgmClient, error) {
 	UUIDs := make([]string, 0)
 	enabledFieldGroup := dcgm.FieldHandle{}
 	requestedFieldIDs := discoverRequestedFieldIDs(config)
-	supportedFieldIDs, err := getAllSupportedFields()
+	supportedRegularFieldIDs, err := getSupportedRegularFields(requestedFieldIDs, logger)
+	if err != nil {
+		// TODO: If there is error querying the supported fields at all, let the
```

Comment:
What's the AI for the TODO? It's not clear whether the rest of the comment is describing the current behaviour or a desired future behaviour.

Reply:
This TODO was actually a reminder for me to revisit this comment, which simply copied the one in lines 81–82. However, your comment made me realize that, unlike in the case of profiling metrics, a failure here would be catastrophic, and should not allow a client to be created. Fixed in https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/pull/219/commits/d0ad1ad8438c58a79e19e4c9eaac0d652eee5043.